examples/shouldi/shouldi/deploy/override/shouldi.yaml

```diff
@@ -27,22 +27,30 @@ definitions:
     name: value
     primitive: generic
 flow:
+  dffml.mapping.create:
+    inputs:
+      key:
+      - seed
+      value:
+      - seed
   lines_of_code_by_language:
     inputs:
       repo:
-      - transform_to_repo: mapping
```

Comment:
Have you run through both tutorials to make sure these changes still result in working tutorials? I'm a bit concerned that we haven't run through both relevant tutorials start to finish.

Reply:
I will try to run through both the tutorials from start to finish and make the changes accordingly.